### PR TITLE
Add device entities

### PR DESCRIPTION
**Pull request by rnestler**
Based on #55

### CODE REVIEW

File: entities/device.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            ' {}\n'.format(self.description) +\
+            ' {}\n'.format(self.keywords) +\
+            ' {}\n'.format(self.author) +\
+            ' {}\n'.format(self.version) +\
+            ' {}\n'.format(self.created) +\
+            ' {}\n'.format(self.deprecated) +\
+            ' {}\n'.format(self.category) +\
+            ' {}\n'.format(self.component_uuid) +\
+            ' {}\n'.format(self.package_uuid)
+        ret += indent_entities(self.pads)
```
Comment (dbrgn) [inline entities/device.py]: We should probably sort the pads first, right?

I'm not sure what sort order LibrePCB uses, but it should be easy to find out.

Comment (rnestler) [reply entities/device.py]: LibrePCB sorts by the Pad UUID.

Comment (rnestler) [reply entities/device.py]: Fixed.

**Comment by rnestler** [comment]
Based on #55, but otherwise good to review.

**Pull request merged**